src/components/Radar/Radar.tsx

```diff
@@ -1,25 +1,24 @@
 import React, {useEffect, useRef, useState} from 'react';
 
-// TODO: Copy the class Radar from 'radar-diagram' and add Typescript there. After that change the Radar component - add Typescript.
-import RadarDiagram from 'radar-diagram';
 import {Tooltip} from 'react-tooltip';
 
 import {useTechContext} from '../../context/Context';
+import RadarDiagram from '../../radar-diagram/radar';
+import {RadarProps} from '../types';
 import './radar.css';
 
 const padding = 50;
 
-const Radar = ({options, segments, rings, elements}) => {
+const Radar = ({segments, rings, elements}: RadarProps) => {
   let svgRef = useRef(null);
-  const [radarDiagram, setRadarDiagram] = useState(
-    new RadarDiagram(options, {elements, rings, segments}),
-  );
+
+  const [radarDiagram, setRadarDiagram] = useState(new RadarDiagram({elements, rings, segments}));
   const {techClicked, setTechClicked} = useTechContext();
 
   useEffect(() => {
-    const radar = new RadarDiagram(options, {elements, rings, segments});
+    const radar = new RadarDiagram({elements, rings, segments});
     setRadarDiagram(radar);
-  }, [options, segments, rings, elements]);
```

Comment:
Nie potrzeba już tych options?

Reply:
Tak, wyrzuciłam to jako, że już mamy swoją klasę radar-diagram i tam do środka wsadziłam te options. Może tak być?

Reply:
👍🏻